source/views.txt

```diff
@@ -0,0 +1,97 @@
+.. _view-details:
+
+=====
+Views
+=====
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Views are read-only results of an
+:ref:`aggregation <compass-agg-builder>` run against a collection.
+|compass-short| displays views in the :ref:`Collections Screen <collection-tab>` with
+a special icon, and indicates the collection from which
+the view was created.
+
+.. figure:: /images/compass/view-list.png
+   :figwidth: 720px
+   :alt: View list
+
+Create a View
+-------------
+
+To create a view, you must use the :ref:`compass-agg-builder`.
+To create a view from your pipeline results:
```

Comment:
nit: There's a bit of a logical jump between "use the agg builder" and "pipeline results". Maybe we could reiterate that the output of the pipeline's final stage becomes the contents of the view?